### DISCUSSION TITLE
Skip build and use image pushed to remote docker

### DISCUSSION THREAD

**Discussion by worm-emoji** [basecamp/kamal#167] [Q&A]
Hi there!

Interested in using mrsk - for a number of reasons we already push a Docker image to `ghcr.io` – is it possible to have mrsk skip the build step and just use an existing image or tag?

**Comment by calmyournerves** [comment]
Yes, the `deploy` command has an option `--skip-push` which skips building and pushing the image. And using `--version`, you can pass the image version that's used. See `mrsk help deploy` for details.
